src/Sulu/Component/Rest/ListBuilder/ListBuilderInterface.php

```diff
@@ -20,10 +20,22 @@
 
     const WHERE_COMPARATOR_UNEQUAL = '!=';
 
+    const WHERE_COMPARATOR_GREATER = '>';
+
+    const WHERE_COMPARATOR_GREATER_THAN = '>=';
+
+    const WHERE_COMPARATOR_LESS = '<';
+
+    const WHERE_COMPARATOR_LESS_THAN = '<=';
+
     const SORTORDER_ASC = 'ASC';
 
     const SORTORDER_DESC = 'DESC';
 
+    const AND_CONJUNCTION = 'AND';
+
+    const OR_CONJUNCTION = 'OR';
```

Comment:
In order to be consistent with the previous constants I would call these `CONJUNCTION_AND` and `CONJUNCTION_OR`